### DISCUSSION TITLE
No fall damage when falling on the hatch

### DISCUSSION THREAD

**Discussion by PlasmaUwU** [FakeFishGames/Barotrauma#13763] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Characters don't take fall damage if they land on hatches

https://github.com/FakeFishGames/Barotrauma/assets/101663742/a2cff4f7-f07e-4b30-a72e-83443856bbdf

### Reproduction steps

1. Make a high sub with hatches instead of a floor
2. Teleport to the top of this sub
3. Fall down
4. Do not take damage from falling

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.3.0.2

### Which operating system did you encounter this bug on?

Windows

**Comment by cl0ck-byte** [comment]
Outposts are also a subject of this bug - very handy though if the generation decides to make a ladder to the heaven.

**Comment by Regalis11** [comment]
This has been addressed in our private repo now, fix coming up in the next unstable build and the upcoming release.
